### ISSUE TITLE
Add keyboard shortcuts to play/pause, next and prev

### ISSUE THREAD

**Issue by AngelEzquerra** [twolfson/google-music-electron#15]
It would be really great if there was a way to configure keyboard shortcuts to play/pause in particular, and also to go to the next or the previous songs.

**Comment by twolfson** [comment]
We bind to media key shortcuts when the application is loaded. Are you receiving any errors in your terminal about them?

https://github.com/twolfson/google-music-electron/blob/1.19.0/lib/shortcuts.js

**Comment by AngelEzquerra** [comment]
The problem is that I am running google music electron on a laptop which has no media buttons. It would be nice to be able to customize these bindings, and, for example, bind play/pause to Left Windows + Alt + P or something like that.

In fact I miss some sort of "options" dialog where you could set these sort of things.

**Comment by twolfson** [comment]
That seems like a reasonable request. I think an options dialog would definitely be the right place.

**Comment by AngelEzquerra** [comment]
In the meantime I changed the shortcuts.js file on my own, but I still think that would be a nice enhancement. For anyone intersted I just did:

```
// Load in our dependencies
var globalShortcut = require('global-shortcut');

// Define a function to bind shortcuts
exports.init = function (gme) {
  // Set up media keys
  //if (!globalShortcut.register('mediaplaypause', gme.controlPlayPause)) {
  if (!globalShortcut.register('ctrl+alt+p', gme.controlPlayPause)) {
    gme.logger.warn('Failed to bind `mediaplaypause` shortcut');
  }
  //if (!globalShortcut.register('medianexttrack', gme.controlNext)) {
  if (!globalShortcut.register('ctrl+alt+.', gme.controlNext)) {
    gme.logger.warn('Failed to bind `medianexttrack` shortcut');
  }
  //if (!globalShortcut.register('mediaprevioustrack', gme.controlPrevious)) {
  if (!globalShortcut.register('ctrl+alt+,', gme.controlPrevious)) {
    gme.logger.warn('Failed to bind `mediaprevioustrack` shortcut');
  }
};
```

**Comment by twolfson** [comment]
Going to look into setting up config management. Thinking about `configstore` but will have to see how friendly that is with `electron's` async main/browser interaction.

https://github.com/yeoman/configstore

**Comment by twolfson** [comment]
This has been implemented in `1.22.0`. On Windows, the config window can be accessed via: "File -> Options".

**Comment by AngelEzquerra** [comment]
Very nice. It works great. Thanks!
